### PR TITLE
Allow BMP, and some other types to be processed for variants

### PR DESCRIPTION
**Pull request by eanders**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

[Attempting to prevent this deprecation warning.](https://green-river.sentry.io/issues/5327260605/?alert_rule_id=12523843&alert_type=issue&environment=production&notification_uuid=862b82fc-cd81-4bf3-8e0d-cdcdf8805664&project=4503977346662400&referrer=slack)

## Type of change
[//]: # 'remove options that are not relevant'
- [x] Code clean-up / housekeeping

## Checklist before requesting review
- [ ] I have performed a self-review of my code
- [ ] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [ ] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [ ] My code follows the style guidelines of this project (rubocop)
- [ ] I have updated the documentation (or not applicable)
- [ ] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -51,6 +51,9 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :delayed_job
     config.action_mailer.deliver_later_queue_name = :mailers
 
+    config.active_storage.variant_processor = :mini_magick
+    config.active_storage.variable_content_types = %w(image/png image/gif image/jpeg image/tiff image/bmp image/webp image/avif image/heic image/heif)
```
Comment (ttoomey) [inline config/application.rb:55]: this is probably the change right?

**Review by ttoomey — COMMENTED** [review]
@eanders it looks like there's a lot of other changes in this PR. Should we merge r15 up into our working release branch in a separate commit?

File: config/application.rb (hunk 1 above)
Comment (eanders) [reply config/application.rb:55]: Yes, this is the change. 
That's so weird, not sure what happened.  I'm going to close and remake the PR.

**Comment by eanders** [comment]
PR is borked.

**Pull request closed without merging**